HouseRules_Core/README.md

```diff
@@ -29,30 +37,61 @@ If installation instructions were followed (i.e., MelonLoader was installed),
 the following file will appear in the Demeo game directory:
 `UserData/MelonPreferences.cfg`.
 
+Demeo should not be running when you are editing this file or your changes may not be saved.
+
 After RulesAPI is installed and Demeo is run at least once, the following will
 appear somewhere in that configuration file:
 
 ```toml
-[RulesAPI]
+[HouseRules]
 ruleset = ""
+loadFromConfig = false
 ```
 
-Select the ruleset to use by typing its name within the quotes.  Alternatively,
-use empty quotes `""` to specify no ruleset should be used.
+A list of out-of-the-box Ruleset names can be found in the
+[Rules readme](../HouseRules_Essentials/README.md).
 
-For now, Demeo should not be running when the configuration file is modified.
-Else your changes may not be saved.
+To use a built-in ruleset, ensure `loadFromConfig` is set to `false` and select the built-in ruleset to use by typing its name within the quotes.
+Alternatively, use empty quotes `""` to specify no ruleset should be used.
+
+Rulesets may also be defined in JSON within `UserData/MelonPreferences.cfg`. Here's a ruleset for beefing up the Sorcerer's Zap ability.
+
+```toml
+[HouseRules]
+ruleset = "SuperZapRuleset"
+loadFromConfig = true
+
+[SuperZapRuleset]
+name = "SuperZapRuleset"
+description = "Zap has no action cost and does 1 extra damage."
+rules = '''
+[
+  {
+    "Rule": "AbilityActionCostAdjustedRule",
```

Comment:
Another tangential thought, not related to the doc:  Now that we've got a good collection of rules, I wonder if we should go back and try to unify and/or improve naming.  I.e.: `AbilityActionCost[[Adjusted]]Rule` -> `Ignored/Waived/Dismissed/Free`.

Reply:
This rule can also be used to add an action cost where there currently isn't one. You could make strenght/speed potions cost to cast if desired. Maybe `Toggle/Switch` instead of `Adjusted`, but I don't really see anything wrong with Adjusted in particular instance. I'll keep the thought in mind whilst I'm looking at things though... There may be other opportunites for improving our naming.